### DISCUSSION TITLE
GNOME Arch Update Extension

### DISCUSSION THREAD

**Discussion by theosmith0308** [wezterm/wezterm#5838] [General]
Gnome comes with Console by default. The command used by Consle to update is kgx -e '/bin/sh -c "sudo pacman -Syu ; echo Done - Press Ctrl + q to exit; read _" ' Now ok. I have already set Wezterm as my default terminal app in Gnome. I now need to know what command to use in the Gnome Arch Update Extension under settings to use Wezterm for the update. If someone can please figure this out you will be amy hero.

**Comment by anasgets111** [comment]
change "kgx" in that line to "wezterm" (in case you didn't figure it out by now or someone else is searching for this)
